CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 0.13.0
+
+## Features
+
+- Commands from actions run in a shell by default; adding `bash -c` is no
+  longer required.
+- The name of the branch is included when updating the `release` field in
+  SPEC-files.
+- Packit configuration can be checked using the new `validate-config`
+  subcommand.
+- If possible, use the subject lines of the commits since the last tag when
+  updating `%changelog`.
+- Use patch metadata when generating patch files for dist-git.
```

Comment:
This needs more details.

Reply:
This is all the details that was in the commit message and there is no user facing documentation that could be linked. We discussed with @lachmanfrantisek that this is used only by us atm. Should I rather not mention it?

Reply:
oki, sounds good to me, feel free to leave it as it is